app/tests/utils/main.cpp

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2019 Oticon A/S
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+// #include <f_core/os/c_datalogger.h>
```

Comment:
OMG ZTests finally? Also dont know if Oticon A/S wrote this

Reply:
Finally got there. And yea Oticon A/S is my side gig from when im not working at nordic yk